app/routes/api/release.js

```diff
@@ -117,7 +118,8 @@ router.post('/:slug', function(req, res) {
           if (releases.length > maxDevReleases) {
             let toSave = [];
             while (toSave.length < maxDevReleases) {
-              toSave.push(releases.pop());
+              //toSave.push(releases.pop());
+              toSave = toSave.concat([releases.pop()]);
```

Comment:
Do we want to remove the commented out lines here?

Reply:
I believe these were holdovers from some early bug fixes I was doing and they didn't end up actually fixing anything. So we can go back to the commented line and remove the one below it or we can just delete the commented lines. I think it's six of one, half-dozen of the other.